packages/editor-ui/src/components/ParameterInputHint.vue

```diff
@@ -1,7 +1,7 @@
 <template>
 	<n8n-text size="small" color="text-base" tag="div" v-if="hint">
-		<div v-if="!renderHTML" :class="{[$style.hint]: true, [$style.highlight]: highlight}">{{ hint }}</div>
-		<div v-else ref="hint" :class="{[$style.hint]: true, [$style.highlight]: highlight}" v-html="sanitizeHtml(hint)"></div>
+		<div v-if="!renderHTML" :class="{[$style.hint]: true}">{{ hint }}</div>
+		<div v-else ref="hint" :class="{[$style.hint]: true}" v-html="sanitizeHtml(hint)"></div>
```

Comment:
https://www.notion.so/n8n/P1-Reviews-Expression-editor-e049831cb7b34f3d8c54bc6cddb522e8#c8171ae8a3784327bff8bf368f486faf

Reply:
I am confused.. are you sure this is what Max wanted here? to remove highlight completely? I think he wanted to fix issue when switching.. 
also if you are going to remove this here, you need to complete the clean up.. change class back to `class="hint"` and remove the prop and where it gets passed

Reply:
When he flagged, he identified these as two issues color switch and `e.g.`. I commented on both and he confirmed, see thread. Else we confirm again?
<img width="694" alt="image" src="https://user-images.githubusercontent.com/44588767/204781049-4c295e5d-a738-48c9-9635-bf8348eeca9e.png">

Reply:
reading through the thread but does not seem to be saying remove it.. please double check

Reply:
/cc @maxtkacz

Reply:
Reverted, see [66570b6](https://github.com/n8n-io/n8n/pull/4631/commits/66570b687581c098915f743666851b1b6ca0e099)